### DISCUSSION TITLE
Ip camera RTMP endpoint not working

### DISCUSSION THREAD

**Discussion by dhsont** [ant-media/Ant-Media-Server#5546] [Q&A]
**Short description**
I added ip camera to ant media server and i can see live stream in ant media server./applications/LiveApp

Now i want to see this stream on youtube so i added youtube RTMP endpoints (rtmp://a.rtmp.youtube.com/live2/j857-j6xa-ss1e-youtube-key) using Add RTMP Endpoint option in antmedia
But video is not streaming to youtube.

but if i create Playlist in antmedia and add a video link (VOD) and Add RTMP Endpoint of youtube it is showing stream on youtube.

means rtmp streaming is working for playlist but not working for ip camera.

**Environment**
Operating system and version: Linux
Ant Media Server version: Community Edition 2.5.2 20230105_1947
Browser name and version: Chrome
Steps to reproduce
add ip camera to antmedia server
add rtmp endpoint of youtube

**Comment by Mohit-3196** [comment]
Hi @dhsont ,
How are you doing?

Please upgrade your Ant Media Server community edition to the latest v2.6.4 from v2.5.2 first.
Here's how you can upgrade:
https://antmedia.io/docs/guides/installing-on-linux/upgrading-ant-media-server/

Once you have upgraded and if this still doesn't work, kindly share your AMS web panel login details with me through support@antmedia.io and I will check it.

Let me know how it goes.
Thank you

**Comment by hemantbhr** [comment]
@dhsont i can help you with this please connect with me on whatsapp +919907099989

**Reply by Mohit-3196** [reply]
Hi @dhsont ,
How are you?

WhatsApp is not our official support channel of so it would be great if you could provide these details to me at mohit.dubey@antmedia.io or you can also write to me at support@antmedia.io by mentioning this thread.

Thank you.

**Reply by Mohit-3196** [reply]
Hi @dhsont ,
How are you doing?

Please let me know if you had a chance to check my message.
I'm waiting for your input.

Thank you.
